test/verify/check-packagekit

```diff
@@ -1035,19 +1035,62 @@ class TestAutoUpdates(PackageCase):
         m.start_cockpit()
         b.login_and_go("/updates")
 
-        # automatic upgrades are not visible
+        # switch to enable automatic upgrades is not visible
         b.wait_present("#available")
-        self.assertFalse(b.is_present("#automatic"))
+        b.wait_present("#automatic")
+        self.assertFalse(b.is_present("#automatic div.btn-onoff-ct"))
 
         # apply updates
-        b.click("#app .container-fluid button")
+        b.click("#app .container-fluid button.pk-update--all")
 
         # empty state visible in main area
         b.wait_present(".container-fluid div.blank-slate-pf")
         self.assertFalse(b.is_present("#available"))
-        # automatic updates are not visible
-        self.assertFalse(b.is_present("#automatic"))
+        # switch to enable automatic upgrades is not visible
+        self.assertFalse(b.is_present("#automatic div.btn-onoff-ct"))
+
+    @skipImage("No supported auto update backend", "centos-7", "debian-stable", "debian-testing", "ubuntu-1604", "ubuntu-stable", "rhel-7-5")
+    def testInstall(self):
+        b = self.browser
+        m = self.machine
+
+        m.execute('dnf remove -y dnf-automatic')
+
+        # provide minimal content in order for the backend to be seen as supported
+        timerContent = '''
+        [Unit]
+Description=dnf-automatic timer
+# See comment in dnf-makecache.service
+ConditionPathExists=!/run/ostree-booted
+
+[Timer]
+OnBootSec=1h
+OnUnitInactiveSec=1d
+
+[Install]
+WantedBy=basic.target
+        '''
+        self.createPackage('dnf-automatic', '1', '1', content = {
+            '/etc/dnf/automatic.conf': '',
+            '/usr/lib/systemd/system/dnf-automatic.timer': timerContent,
+            '/usr/lib/systemd/system/dnf-automatic-install.timer': timerContent
+        });
+        self.enableRepo()
 
+        m.start_cockpit()
+        b.login_and_go('/updates')
+
+        # click through install dialog
+        b.wait_present('#automatic')
+        b.click('#automatic button.btn')
```

Comment:
This needs to wait for the button to be present. (Also, it should be the On/Off switch, see above).